test-apps/display-performance-test-app/src/frontend/DisplayPerformanceTestApp.ts

```diff
@@ -26,6 +26,13 @@ export class DisplayPerfTestApp {
     iModelApp = iModelApp ?? {};
     iModelApp.i18n = { urlTemplate: "locales/en/{{ns}}.json" } as I18NOptions;
 
+    /** eslint-disable @typescript-eslint/naming-convention */
+    iModelApp.mapLayerOptions = {
+      MapBoxImagery: process.env.SVT_MAPBOX_KEY ? { key: "access_token", value: process.env.SVT_MAPBOX_KEY } : undefined,

```

Comment:
@hnn0003 I noticed that you're not using env vars at all for dpta, would you want me to add this to the config.json values allowed? Or is there a better way?

Reply:
@hnn0003 @DStradley probably better if everything's in the config, right? As long as you don't commit a config containing the keys.

Reply:
Didn't see this till now. Yeah, we generally are trying not to use env variables in dpta, so it seems like having an option for this in the config json file would be better. Can we add a default value of what this would be to the dpta code, though (the actual .ts code--not a committed config file)? That way, the auto tests and any older tests would still be able to function without any retro changes?

Reply:
No, we want to avoid publicly exposing any secrets going forward so it would always need to be configured whenever you're running the app.

Reply:
I think @hnn0003 would like you to privately inform us of what keys we should be using.

Reply:
@pmconne I think config will be necessary for some situations, but it seems like in others it would be useful as an environment var (set it for a machine and forget it, rather than each config).  Could we do where either works?
Also, I assume the yaml and other testing stuff where this key will have to go now is not public?

Reply:
Okay, I'll support both and push that up in a few minutes.

The YAML pipelines will need to set these variable in a Variable Group and no they aren't exposed publicly. The builds are internal.

Reply:
If env var will suffice for now, that would be preferable to making substantial changes to dpta as part of this PR.

Reply:
Just pushed the update to the README. I'll send an email with updated keys.